src/Components/Admin/admin.js

```diff
@@ -331,31 +352,35 @@ export default function Admin() {
                         anchorEl={anchorEl}
                         anchorOrigin={{
                             vertical: "top",
-                            horizontal: "center",
+                            horizontal: -30,
                         }}
                         transformOrigin={{
-                            vertical: "top",
+                            vertical: 10,
                             horizontal: "center",
                         }}
+                        MenuListProps={{ disablePadding: true }}
                         keepMounted
                         open={open}
                         onClose={handleAddButtonClose}
                     >
+                        <p className={classes.menuItemTitle}>New</p>
```

Comment:
@andstun - Regarding the comment I gave you on your PR about formatting the "title" of the drop down menu. You can do something like this. See what the styling is in "menuItemTitle" above and how it is used here (`<p>` tag).